src/resources/EnplugDetailsRealTimeContext.js

```diff
@@ -0,0 +1,46 @@
+import RealTimeContext from './RealTimeContext';
+
+/**
+ * A real-time context that can be used to generate subscriptions to enplug details for vehicles
+ */
+class EnplugDetailsRealTimeContext extends RealTimeContext {
+  /**
+   * Creates a context that can subscribe to EnplugDetails updates.
+   * @param {RealTimeClient} realTimeClient Pre-configured instance of RealTimeClient
+   * @param {string} customerCode Alphanumeric code of the customer
+   */
+  constructor(realTimeClient, customerCode) {
+    const entityName = 'ENPLUG_DETAILS';
+    super(realTimeClient, entityName, customerCode);
+    this.filters = {
+      vehicles: [],
+    };
+  }
+
+  /**
+   * Restrict subscriptions created by this context to a single vehicle.
+   * Overwrites any existing vehicle filter for this context.
+   * Cannot be called after subscription is started.
+   * @param {Resource|string} vehicle Href or resource representation of a Vehicle
+   * @returns {EnplugDetailsRealTimeContext} Context with filter applied.
+   */
+  forVehicle(vehicle) {
+    return this.forVehicles([vehicle]);
```

Comment:
This needs a resolveHref call I think

Reply:
@cuevaskoch  the call on line 41 should take care of it right? I am not seeing any problems on my end and I mostly boiler plated this over from Vehicles.

Reply:
Whoa, I must have misread.  Sorry!